pipeline_dp/budget_accounting.py

```diff
@@ -85,7 +91,9 @@ class BudgetAccountant(abc.ABC):
     def request_budget(self,
                        noise_kind: NoiseKind,
                        sensitivity: float = 1,
-                       weight: float = 1) -> MechanismSpec:
+                       weight: float = 1,
+                       count: int = 1,
+                       noise_standard_deviation: float = 1) -> MechanismSpec:
```

Comment:
Nit: float -> Optional[float] = None (and in other places)

Reply:
Done